cli/tests/pcluster/config/test_validators.py

```diff
@@ -97,11 +98,15 @@ def test_ec2_instance_type_validator(mocker, instance_type, expected_message):
         ("awsbatch", "c4.xlarge", "is not supported"),
         ("awsbatch", "t2", None),  # t2 family
         ("awsbatch", "optimal", None),
+        ("sge", "p4d.24xlarge", "has 4 Network Interfaces."),
```

Comment:
let's add a test for slurm to be sure it's passing.

Reply:
Done